### PR TITLE
Project modal and popup feature

### PR DESCRIPTION
**Pull request by jash-debug**
- Add modal js structure
- Add popup feature structure
- Design using classes
- Add images 
- implement desktop and mobile versions
- Fix linter errors
- delete commented code

### CODE REVIEW

File: style.css [hunk 1]
```diff
@@ -302,6 +376,17 @@ li:nth-child(1) {
...
+.projectPopup {
+  display: flex;
+  flex-direction: column;
+  width: 95vw;
+  height: 90vh;
+  align-items: center;
+  justify-content: center;
+  background-color: #cfd8dc;
+} */
+
```
Comment (ahmetbozaci) [inline style.css]: - [ ] To make clean code, kindly delete commented code. [Requirement](https://github.com/microverseinc/curriculum-html-css/blob/main/articles/html_css_best_practices.md#:~:text=Do%20not%20commit%20old%20pieces%20of%20code%20as%20inline%20comments.%20They%20will%20make%20your%20project%20look%20messy.%20If%20you%20need%20to%20review%20a%20previous%20version%20of%20your%20code%2C%20you%20can%20always%20use%20git%20history.)

File: style.css [hunk 2]
```diff
@@ -647,7 +722,7 @@ button:active {
     align-self: center;
   }
 
-  .popup {
+  /* .popup {
```
Comment (ahmetbozaci) [inline style.css]: - [ ] To make clean code, kindly delete commented code. [Requirement](https://github.com/microverseinc/curriculum-html-css/blob/main/articles/html_css_best_practices.md#:~:text=Do%20not%20commit%20old%20pieces%20of%20code%20as%20inline%20comments.%20They%20will%20make%20your%20project%20look%20messy.%20If%20you%20need%20to%20review%20a%20previous%20version%20of%20your%20code%2C%20you%20can%20always%20use%20git%20history.)

**Review by ahmetbozaci — CHANGES_REQUESTED** [review]
# ♻️♻️♻️ STATUS: Changes Required  ♻️♻️♻️


 @jash-debug  ✋🏼 

#### Your project is great. You are almost done. However, there are some issues you should fix. Please fix them then your project will be approved.

___ 

### 💯  Highlights 💯
- Your git-flow is correct. ✔️
- You set up linter files and no errors. ✔️
- Your pull request is very professional. It has a meaningful title and summary. Keep it up.✔️
- You create a popup with javascript and it works well.✔️
- You follow the design
___


### ♻️ Request changes ♻️

- [ ] _Check the comments under the review._

___

### Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._
___

### Cheers and Happy coding! 💻 👏
almost

___

_Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification._

_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._


___

_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form.](https://airtable.com/shrQAqnBwek5a0O0s)_

[GitHub](https://github.com/ahmetbozaci) | [LinkedIn](https://linkedin.com/in/ahmetbozaci) | [Twitter](https://twitter.com/ahmtbozaci)

_Reviewed by [Ahmet Bozacı ](https://github.com/ahmetbozaci)_

**Review by zhadier — APPROVED** [review]
## Hi @jash-debug  👋  

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 



# Congratulations! 🎉 

✔️ No linter errors
✔️ Professional pull request title and description

### Keep Rocking!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**